src/did/src/permission.rs

```diff
@@ -18,6 +18,8 @@ pub enum Permission {
     UpdateLogsConfiguration,
     /// Allows caller to reset the EVM state
     ResetEvmState,
+    /// Allows the caller to send transactions to the EVM
```

Comment:
Please refine the description and maybe the name. Everyone is allowed to send transaction, so what does this permission give in addition to that?